cmd/nex/workload_test.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func initWorkloadCommand(t testing.TB) (string, error) {
+	t.Helper()
+
+	confDir := t.TempDir()
+	f, err := os.Create(filepath.Join(confDir, "publisher.xk"))
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	f, err = os.Create(filepath.Join(confDir, "issuer.nk"))
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	wl, err := os.Create(filepath.Join(confDir, "workload"))
+	if err != nil {
+		return "", err
+	}
+	defer wl.Close()
+	_, err = wl.Write([]byte("test"))
+	if err != nil {
+		return "", err
+	}
+
+	return confDir, nil
+}
+
+func TestStartWorkload(t *testing.T) {
+	confDir, err := initWorkloadCommand(t)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	w := Workload{
+		Run: RunWorkload{
+			Devrun:       false,
```

Comment:
We should remove this flag and have tests asserting all the things that need defaults are set if they aren't provided.